examples/remarketing/add_customer_match_user_list.py

```diff
@@ -249,6 +242,66 @@ def _build_offline_user_data_job_operations(client):
     ]
 
 
+def _check_job_status(
+    client,
+    customer_id,
+    offline_user_data_job_resource_name,
+    user_list_resource_name,
+):
+    """Retrieves, checks, and prints the status of the offline user data job.
+
+    Args:
+        client: The Google Ads client.
+        customer_id: The customer ID for which to add the user list.
+        offline_user_data_job_resource_name: The resource name of the offline
+            user data job to get the status of.
+        user_list_resource_name: The resource name of the customer match user
+            list
+    """
+    job_status_enum = client.get_type(
+        "OfflineUserDataJobStatusEnum", version="v6"
+    ).OfflineUserDataJobStatus
+    job_type_enum = client.get_type(
+        "OfflineUserDataJobTypeEnum", version="v6"
+    ).OfflineUserDataJobType
+    query = f"""
+        SELECT
+          offline_user_data_job.resource_name,
+          offline_user_data_job.id,
+          offline_user_data_job.status,
+          offline_user_data_job.type,
+          offline_user_data_job.failure_reason
+        FROM offline_user_data_job
+        WHERE offline_user_data_job.resource_name =
+          '{offline_user_data_job_resource_name}'
+        LIMIT 1"""
+
+    # Issues a search request using streaming.
+    google_ads_service = client.get_service("GoogleAdsService", version="v6")
+    results = google_ads_service.search(customer_id, query=query)
+    offline_user_data_job = next(iter(results)).offline_user_data_job
+    status = offline_user_data_job.status
+    status_name = job_status_enum.Name(status)
+
+    print(
+        f"Offline user data job ID '{offline_user_data_job.id}' with type "
+        f"'{job_type_enum.Name(offline_user_data_job.type)}' has status: "
+        f"{status_name}"
+    )
+
+    if status_name == "SUCCESS":
+        _print_customer_match_user_list_info(
+            client, customer_id, user_list_resource_name
+        )
+    elif status_name == "FAILED":
+        print(f"\tFailure Reason: {offline_user_data_job.failure_reason}")
+    elif status_name in ("PENDING", "RUNNING"):
+        print(
+            "To check the status of the job periodically, use the following "
+            f"GAQL query with GoogleAdsService.search: {query}"
```

Comment:
Capitalize `Search`

Reply:
Or maybe make it google_ads_service.search for consistency with above. These are nits though